src/pages/api/places/index.js

```diff
@@ -0,0 +1,24 @@
+import dbConnect from "../../../../lib/connect";
+import Place from "../../../../lib/models/Place";
+
+export default async function handler(request, response) {
+  await dbConnect();
+
+  if (request.method === "GET") {
+    const places = await Place.find();
+
+    return response.status(200).json(places);
+  }
+
+  if (request.method === "POST") {
```

Comment:
In general, i think our database queries have the potential to fail for reasons outwith our control. For that reason it is generally best to wrap these requests (not just the POST) in a `try catch`. 

[Example from notes](https://github.com/neuefische/bo-web-23-2/blob/main/sessions/backend-create/backend-create.md#create-with-mongoose)